examples/hello_colors/src/app.rs

```diff
@@ -30,8 +30,6 @@ impl App {
 pub fn init() -> Result<(), eframe::Error> {
     let options = eframe::NativeOptions {
         viewport: egui::ViewportBuilder::default().with_maximized(true),
-        follow_system_theme: false,
-        default_theme: eframe::Theme::Light,
```

Comment:
These options are gone, not sure what to do here

Reply:
Thanks! Have just committed the changes for egui 0.29.0 and published 0.3.0
Please let me know if I have missed anything.